docs/developers/reference/api/linea-gettransactionexclusionstatusv1.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: linea_getTransactionExclusionStatusV1
+description: Reference content for the linea_getTransactionExclusionStatusV1 method.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# `linea_getTransactionExclusionStatusV1`
+
+Queries a database maintained by the transaction exclusion API service to check if a transaction 
+was rejected because it exceeded data line limits that would prevent the prover from generating a 
+proof for the block. The database is updated by a parallel write API endpoint called by the 
+sequencer, edge nodes, or RPC nodes.
+
+If the transaction was rejected, the API call will be successful and provide the reason. 
+
+Transactions that were not rejected for this reason will return `null` as the `result`. 
+
+:::warning
+Transactions older than 24 hours cannot be checked.
+:::
+
+## Parameters
+
+`txHash`: [Required] The hash of the subject transaction.
+
+## Returns
+
+- `txHash`: The hash of the subject transaction. 
+- `from`: The public address of the account that sent the transaction. 
+- `nonce`: The transaction nonce, hexadecimal. 
+- `txRejectionStage`: A string detailing the point at which the transaction was rejected. One of:
+    - `SEQUENCER`: Rejected by the sequencer.
+    - `RPC`: Rejected by an RPC node (e.g. `https://rpc.linea.build`).
+    - `P2P`: Rejected by a P2P-connected node (e.g. an edge node). 
+- `reasonMessage`: Explains why the transaction was rejected. 
+- `blockNumber`: The block that the transaction was rejected from, in hexadecimal format. Only 
+returned for transactions rejected by the sequencer. 
+- `timestamp`: Time of rejection, in ISO8601 format. 
+
+## Example
+
+### Request
+
+<Tabs>
+    <TabItem value="cURL">
+        ```bash
+        curl https://rpc.linea.build \
```

Comment:
no key required -- this is public RPC with no denial of service structure?

Reply:
good point. Following up separately and will return and amend if needed.